features/support/content_register.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../../test/support/content_register_helpers'
+
+Before do
```

Comment:
That would mean having to add the tag to every feature that involves loading the edition form on an edition type that supports policies as they will all attempt to load policies from the content register.